airbyte-integrations/connectors/source-slack/source_slack/source.py

```diff
@@ -21,7 +21,7 @@
 class SlackStream(HttpStream, ABC):
     url_base = "https://slack.com/api/"
     primary_key = "id"
-    page_size = 1000
+    page_size = 100
```

Comment:
@davydov-d can we add a note explaining the reason of choosing 100 and why we're not going to change? I'm saying this because in Slack API and from our tests increasing to 1000 works.

Reply:
Sure, will do